pkg/appfile/appfile.go

```diff
@@ -352,65 +325,59 @@ func makeWorkloadWithContext(pCtx process.Context, wl *Workload, ns, appName str
 	return workload, nil
 }
 
-// evalWorkloadWithContext evaluate the workload's template to generate component and ACComponent
```

Comment:
you should keep and update the function annotation

Reply:
fixed.